### PR TITLE
chore: generated project structure using repo-config

### PR DESCRIPTION
**Pull request by sandovalrr**
- Generate project using repo-config
- Remove example files

### CODE REVIEW

File: .cookiecutter-replay.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{
+  "cookiecutter": {
+    "Introduction": "",
+    "type": "api",
```
Comment (cwasicki) [inline .cookiecutter-replay.json]: This should be `lib` type. You can also compare here: https://github.com/frequenz-floss/frequenz-client-microgrid-python/blob/v0.x.x/.cookiecutter-replay.json

File: .cookiecutter-replay.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    "name": "frequenz-client-reporting-python",
```
Comment (cwasicki) [inline .cookiecutter-replay.json]: Name would be `client-reporting` (according to the linked file).

File: proto/frequenz/api/frequenz_client_reporting_python/frequenz_client_reporting_python.proto [hunk 3]
```diff
@@ -0,0 +1,23 @@
+// Frequenz Reporting API Client
```
Comment (cwasicki) [inline proto/frequenz/api/frequenz_client_reporting_python/frequenz_client_reporting_python.proto]: This file should not be here.

File: py/frequenz/client/reporting/__init__.py [hunk 4]
```diff
@@ -0,0 +1,7 @@
+# License: MIT
```
Comment (cwasicki) [inline py/frequenz/client/reporting/__init__.py]: The whole `py` folder should also not be here.

**Review by cwasicki — CHANGES_REQUESTED** [review]
There are some files related to API. Looks to me like you have to re-generate the whole repo with correct type and name and commit only that output.

**Comment by sandovalrr** [comment]
> There are some files related to API. Looks to me like you have to re-generate the whole repo with correct type and name and commit only that output.

sure, let me do that

**Comment by cwasicki** [comment]
@sandovalrr Can you confirm whether you committed the auto-generated files as-is?

**Pull request closed without merging**